src/async/posix_event_loop.zig

```diff
@@ -755,21 +754,30 @@ pub const FilePoll = struct {
         this.deactivate(event_loop_ctx.platformEventLoop());
     }
 
-    pub fn onTick(loop: *Loop, tagged_pointer: ?*anyopaque) callconv(.C) void {
+    pub fn onTick(loop: *Loop, tagged_pointer: ?*anyopaque) callconv(.C) bool {
```

Comment:
can you make this boolean an enum so we don't need to guess what it means for onTick to return true or return false